data-reporting-service/post-processing-service/src/main/resources/application.yaml

```diff
@@ -2,8 +2,6 @@ spring:
   kafka:
     topic:
       investigation: nrt_investigation
-      investigation-confirmation: nrt_investigation_confirmation
-      investigation-notification: nrt_investigation_notification
       observation: nrt_observation
       organization: nrt_organization
       patient: nrt_patient
```

Comment:
Consider Updating the port number from 8090 to something different. We are using the same port for the other services too.  Prod deployment won't be an issue but locally running multiple services on the same port might throw some issues.

Reply:
> Consider Updating the port number from 8090 to something different. We are using the same port for the other services too. Prod deployment won't be an issue but locally running multiple services on the same port might throw some issues.

Just FYI, the port 8093 is taken for investigation preprocessing service (the PR under review)

Reply:
when running in local, this is always going to be an issue. so, let's leave it this way and user can fix when running it in local